project/PlayCrossCompilation.scala

```diff
@@ -0,0 +1,4 @@
+import uk.gov.hmrc.playcrosscompilation.AbstractPlayCrossCompilation
+import uk.gov.hmrc.playcrosscompilation.PlayVersion._
+
+object PlayCrossCompilation extends AbstractPlayCrossCompilation(defaultPlayVersion = Play26)
```

Comment:
Can we make defaultPlayVersion = Play25?

Reply:
Would it be possible to add on this change post PR merge?

Reply:
@jmountain-hmrc The current default for libraries is Play 2.5.  In your microservice you should override it to produce a Play 2.6 artifact.  Please check https://github.com/hmrc/sbt-play-cross-compilation/blob/master/README.md.

Reply:
@jmountain-hmrc The current default for libraries is Play 2.5 as it is the current version used on the Platform. I think it will cause less confusion since it is the version sbt and your IDE will pick up.

Reply:
Done